scvi/models/vae.py

```diff
@@ -232,6 +233,9 @@ def inference(self, x, batch_index=None, y=None, n_samples=1):
             library=library,
         )
 
+    def compute_global_kl_divergence(self):
+        return 0.0
```

Comment:
same here ?

Reply:
We do call it in `compute_elbo` in its current refactorized form